bot/client.js

```diff
@@ -57,4 +57,12 @@ client.on('message', (message) => {
   }
 });
 
+// Listen for Guild Member Add event
+client.on('guildMemberAdd', (member) => {
```

Comment:
Narrow logic that does not utilize the existing controller logic. This is a one time use feature while the rest of the framework uses controllers to handle client events.